src/app/launcher/service/project-summary.service.ts

```diff
@@ -12,12 +12,5 @@ export abstract class ProjectSummaryService {
    * @param {Summary} summary The project summary
    * @returns {Observable<boolean>}
    */
-  abstract setup(summary: Summary, spaceId: string, spaceName: string, isImport: boolean): Observable<any>;
-
-  /**
-   * Get the current context details
-   *
-   * @returns {Observable<any>}
-   */
-  abstract getCurrentContext(): Observable<any>;
+  abstract setup(summary: Summary, retry?: number): Observable<any>;
```

Comment:
Is spaceId not needed for `/launch` anymore ?

Reply:
Yes it's still needed, but it would fetch spaceId from fabric8-ui and then pass it again here. So simpler would be to keep it on the fabric8-ui see https://github.com/fabric8-ui/fabric8-ui/pull/3019

Reply:
tried with fabric8-ui/fabric8-ui#3019 looks good!!